content/ansible/roles/manageiq-core.manageiq-vmdb/README.md

```diff
@@ -0,0 +1,121 @@
+manageiq-core.manageiq-vmdb
+=========
+
+The `manageiq-core.manageiq-vmdb` role allows for users of ManageIQ to modify and/or change VMDB objects via an Ansible Playbook.
+The role includes a module `manageiq_vmdb` which does all the heavy lifting needed to modify or change objects in the database.
+
+Requirements
+------------
+
+ManageIQ has to be Gaprindashvili (G Release) or higher.
```

Comment:
Minor, but the REST API actually has a separate versioning scheme, so it's probably better to align to the API version.

Reply:
@Fryguy - I can align the API version - how best to say that here?

Reply:
Maybe @abellotti can advise us here.